src/validator.ts

```diff
@@ -50,7 +50,25 @@ export class Validator {
     }
 
     // Inheritdoc is not enforced nor present, and there is no other documentation, returning error
-    if (!natspec.tags.length) return [`Natspec is missing`];
+    if (!natspec.tags.length) {
+      // If node is a function, check the user defined config
+      if (node instanceof FunctionDefinition) {
+        let needsWarning = false;
+
+        Object.keys(this.config.functions).forEach((key) => {
+          Object.keys(this.config.functions[key as keyof Functions].tags).forEach((tag) => {
+            if (this.config.functions[key as keyof Functions][tag as keyof FunctionConfig]) {
+              needsWarning = true;
+            }
```

Comment:
One thing to note here. We could avoid repeating `as keyof` by converting the config into a class and adding helpers to it. This approach I believe will also help with retrieving the rest of the rules. Let's discuss it later when the logic is fully developed.